api.planx.uk/modules/webhooks/service/sanitiseApplicationData/operations.ts

```diff
@@ -291,3 +294,23 @@ export const deleteHasuraScheduledEventsForSubmittedSessions: Operation =
     const [_column_name, ...ids] = response?.result?.flat() || [];
     return ids;
   };
+
+export const deleteFeedback: Operation = async () => {
+  const mutation = gql`
+    mutation DeleteFeedback($retentionPeriod: timestamptz) {
+      feedback: delete_feedback(
+        where: { created_at: { _lt: $retentionPeriod } }
+      ) {
+        returning {
+          id
+        }
+      }
+    }
+  `;
+  const {
+    feedback: { returning: result },
+  } = await $api.client.request<{ feedback: Result }>(mutation, {
+    retentionPeriod: getRetentionPeriod(),
```

Comment:
_Comment_

I erroneously looked at the `getRetentionPeriod` from `pay`: editor.planx.uk/src/lib/pay.ts which is 28 days 🤦 

At the top of the file it has the period as 6 months. 

I'm wondering whether we stick to 6 months to be consistent with the rest of the sanitisation or swap it to 28 days as originally planned?

Reply:
Yep no problems here - the simpler the better 👍 

Let's make sure we communicate this to George / August / Silvia and update Notion to now reflect the changes made in this PR - https://www.notion.so/opensystemslab/Data-Retention-Policy-fcd47af2a8244888b544a1e9c92c88f8

Reply:
Great shout! I didn't think to look for a data policy on Notion. The other place is https://www.planx.uk/privacy and Alastair has advised I should give him a shout when the broader feature is ready and he'll update this. 

Another place would be the flow setting level privacy policy potentially.